README.md

```diff
@@ -8,6 +8,8 @@
 
 Loggly broadcasting for Good.
 
+`good-loggly` is a [good](https://github.com/hapijs/good) reporter implementation to write [hapi](http://hapijs.com/) server events to Loggly.
```

Comment:
You can leave the example as is. Just clarify that it's not a good specific thing. It's a writable stream interface for writing things to loggly. Then say something like "here is an example of using it to log events from good".

Something along those lines.